releasenotes/notes/create-auth-token-file-only-for-run-cmd-d98a86e00fefaa9f.yaml

```diff
@@ -0,0 +1,6 @@
+enhancements:
+  - |
+    The authentication token file is now only created
+    when the agent is launch with the ``agent run`` command
```

Comment:
nits:
 - `launched`
 - it's better to mention `agent start` than `agent run` (as mentioned in the official doc: https://docs.datadoghq.com/fr/agent/guide/agent-commands/?tab=agentsv6etv7), if I'm right `run` is the deprecated command coming from A5.

Reply:
Thanks ! Never noticed that `agent run` was deprecated in favour of `agent start` because the systemd unit still uses `agent run`...